refarch-gateway/src/test/java/de/muenchen/refarch/gateway/filter/GlobalBackend5xxTo400MapperTest.java

```diff
@@ -30,51 +31,53 @@
+                                new HttpHeader(org.springframework.http.HttpHeaders.EXPIRES, "Wed, 21 Oct 2099 07:28:06 GMT")))
                         .withBody("{ \"testkey\" : \"testvalue\" }")));
 
         webTestClient.get().uri("/api/refarch-gateway-backend-service/remote").exchange()
                 .expectStatus().isEqualTo(HttpStatus.BAD_REQUEST)
-                .expectHeader().valueMatches("Content-Type", "application/json")
-                .expectHeader().doesNotExist("WWW-Authenticate")
-                .expectHeader().valueMatches("Expires", "0")
+                .expectHeader().valueMatches(org.springframework.http.HttpHeaders.CONTENT_TYPE, ContentType.APPLICATION_JSON.getMimeType())
+                .expectHeader().doesNotExist(org.springframework.http.HttpHeaders.WWW_AUTHENTICATE)
+                .expectHeader().valueMatches(org.springframework.http.HttpHeaders.EXPIRES, "0")
                 .expectBody()
                 .jsonPath("$.status").isEqualTo("400")
                 .jsonPath("$.error").isEqualTo("Bad Request");
     }
 
     @Test
     @WithMockUser
-    public void backendError200() {
+    void backendError200() {
 
         stubFor(get(urlEqualTo("/remote"))
                 .willReturn(aResponse()
                         .withStatus(HttpStatus.OK.value())
                         .withHeaders(new HttpHeaders(
-                                new HttpHeader("Content-Type", "application/json"),
-                                new HttpHeader("WWW-Authenticate", "Bearer realm=\"Access to the staging site\", charset=\"UTF-8\""),
-                                new HttpHeader("Expires", "Wed, 21 Oct 2099 07:28:06 GMT")))
+                                new HttpHeader(org.springframework.http.HttpHeaders.CONTENT_TYPE, ContentType.APPLICATION_JSON.getMimeType()),
+                                new HttpHeader(org.springframework.http.HttpHeaders.WWW_AUTHENTICATE,
+                                        "Bearer realm=\"Access to the staging site\", charset=\"UTF-8\""),
+                                new HttpHeader(org.springframework.http.HttpHeaders.EXPIRES, "Wed, 21 Oct 2099 07:28:06 GMT")))
                         .withBody("{ \"testkey\" : \"testvalue\" }")));
 
         webTestClient.get().uri("/api/refarch-gateway-backend-service/remote").exchange()
                 .expectStatus().isEqualTo(HttpStatus.OK)
-                .expectHeader().valueMatches("Content-Type", "application/json")
-                .expectHeader().doesNotExist("WWW-Authenticate")
-                .expectHeader().valueMatches("Expires", "0")
+                .expectHeader().valueMatches(org.springframework.http.HttpHeaders.CONTENT_TYPE, ContentType.APPLICATION_JSON.getMimeType())
+                .expectHeader().doesNotExist(org.springframework.http.HttpHeaders.WWW_AUTHENTICATE)
+                .expectHeader().valueMatches(org.springframework.http.HttpHeaders.EXPIRES, "0")
```

Comment:
Please don't use the FQCN here and instead import correctly. :)

Reply:
See above